### ISSUE TITLE
Javascrip Error with geojsonhint: Unexpected Identifier

### ISSUE THREAD

**Issue by ateucher** [ropensci/geojsonio#73]
I'm getting a javascript error from geojsonhint when linting the `states` dataset converted to `json`:

```r
library(geojsonio)
states_json <- geojson_json(states)
lint(states_json)
## Error: SyntaxError: Unexpected identifier
```

It must be something to do with the way the object is passed into the `V8` environment. I've implemented a fix here: https://github.com/ateucher/geojsonio/commit/4a5fcec389f8f03ae22d8aca6902a2b58d69ac1c. I assigned the `geojson` object to a variable first with `ct$assign`, then run `geojsonhint` on the new variable with `ct$eval`. Don't know why it works any differently, but it does seem to. All automated tests passed for me, and linting `states_json` works. I can do a PR if you think it's okay.

**Comment by ateucher** [comment]
Probably worth makings sure you get the same error I do first though...

**Comment by sckott** [comment]
Thamks. Will take a look in a few hours

**Comment by ateucher** [comment]
Great. No rush

**Comment by sckott** [comment]
Yep, that change makes sense, PR then?

**Comment by ateucher** [comment]
Sent!

**Comment by sckott** [comment]
closed in #74
